### ISSUE TITLE
Tailscale Log in from HA OS returns "Failed to log in: null is not an object (evaluating 'window.open(url, "_blank").focus')"

### ISSUE THREAD

**Issue by outthereandaway** [hassio-addons/addon-tailscale#52]
# Problem/Motivation

When opening the webUI of the started add-on with "OPEN WEB UI" button in Supervisor->Tailscale->Info, the login window is displayed. Upon clicking the "Log In" button, the following error is shown "Failed to log in: null is not an object (evaluating 'window.open(url, "_blank").focus')". The log under Supervisor->Tailscale->Log shows simply "false" on every line.

## Expected behavior

The Log In button should lead to a webpage where the Homeassistant instance can be coupled with the Tailscale account.

## Actual behavior

Upon clicking the "Log In" button, the following error is shown "Failed to log in: null is not an object (evaluating 'window.open(url, "_blank").focus')". The log under Supervisor->Tailscale->Log shows simply "false" on every line.

## Steps to reproduce

Use Homeassistant OS 6.6, core-2021.11.3, supervisor-2021.10.8 on Raspberry Pi 4 8GB RAM with SSD, install Tailscale add-on through Supervisor->Add-on Store, press "START", press "OPEN WEB UI", press "Log In"

## Proposed changes

> (If you have a proposed change, workaround or fix,
> describe the rationale behind it)

**Comment by frenck** [comment]
Which browser are you using?

**Comment by outthereandaway** [comment]
Dear frenck, many thanks for your answer. I am using Safari.

**Comment by frenck** [comment]
Safari on a mobile device?

**Comment by outthereandaway** [comment]
Nope, sorry to be not precise enough. Safari on MacOS 11.6.

**Comment by frenck** [comment]
As a workaround, it should work in other browsers. But indeed, confirmed not working in Safari

**Comment by outthereandaway** [comment]
By the way: Same behaviour using Firefox on MacOS 11.6.

Chrome on MacOS 11.6 works!!

(Sorry, could have had this idea before). Thanks for your valuable help anyway!!

**Comment by thetmtech** [comment]
This issue was resolved for me by allowing popup windows in Safari, macOS 12.0.1.

**Comment by jakesmolka** [comment]
> This issue was resolved for me by allowing popup windows in Safari, macOS 12.0.1.

Same 👍

**Comment by genebean** [comment]
Firefox and Safari on iOS both fail for me. 

Safari:

![2D5C0E5E-A90F-4576-B4BE-89E267C2727A](https://user-images.githubusercontent.com/966789/147714682-da5e1a45-8116-4559-ad1d-fa526006e5df.png)

Firefox: 
![B8FCF0A0-8E6C-4F21-A025-115428D2B62E](https://user-images.githubusercontent.com/966789/147714685-6304197c-997f-4ca2-a78c-fc75247880f5.png)

**Comment by genebean** [comment]
Oh, and the HA Companion app also fails.

**Comment by frenck** [comment]
Mobile will fail. Not much I can do about that.

**Comment by kuduacz** [comment]
Same here.
Mobile Safari and HA companion app

**Comment by Franz-at-Github** [comment]
Experienced the same error when I tried to install and launch Tailscale from my IPad Home-Assistant App.
Fortunately a google search brought me to this page, and I then I logged in to HomeAssistant from my Windows PC, and the start of Tailscale worked well.

Probably would make sense to mention in the „About comment“ that Tailscale  can not be launched directly from the Home-Assistant iOS App. Otherwise people just might give up when it doesn’t work on first attempt.

One issue still remains on my installation (didn’t investigate yet more deep):
Grafana Dashboard is not shown in my Home-Assistant iOS App when I use Tailscale vpn.
